pkg/controllermanager/controller/project/stale/reconciler.go

```diff
@@ -151,9 +150,40 @@ func (r *Reconciler) projectInUseDueToBackupEntries(ctx context.Context, namespa
 	return kubernetesutils.ResourcesExist(ctx, r.Client, &gardencorev1beta1.BackupEntryList{}, r.Client.Scheme(), client.InNamespace(namespace))
 }
 
+func (r *Reconciler) projectInUseDueToWorkloadIdentities(ctx context.Context, namespace string) (bool, error) {
+	workloadIdentityList := &metav1.PartialObjectMetadataList{}
+	workloadIdentityList.SetGroupVersionKind(securityv1alpha1.SchemeGroupVersion.WithKind("WorkloadIdentityList"))
+	err := r.Client.List(
+		ctx,
+		workloadIdentityList,
+		client.InNamespace(namespace),
+		gardenerutils.UncontrolledSecretSelector,
```

Comment:
What is the reason for this selector? For secrets it is useful because they are generic resource used in multiple cases and `projectInUseDueToSecrets` probably want to take a short-cut here and filter out early the secrets in the project namespace managed by gardener, e.g. this matching the selector `gardener.cloud/role notin (kubeconfig, ssh-keypair, monitoring)`.

As there is no gardener managed workloadidentities, I propose to remove the selector from the list call

```suggestion
```

Reply:
Copy/paste mistake. Will remove the selector as it is not needed.